consensus-types/types/validator.go

```diff
@@ -364,6 +364,5 @@ func (v *Validator) Status(currentEpoch math.Epoch) (string, error) {
 		}
 		return constants.ValidatorStatusWithdrawalDone, nil
 	}
-
```

Comment:
super nit: consider removing this (unnecessary diff)

Reply:
done!